### PR TITLE
EPMRPP-83970 || Update the Filter on Monitoring page with new events

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/common/constants/localization/eventsLocalization.js [hunk 1]
```diff
@@ -123,13 +122,9 @@ export const actionMessages = defineMessages({
...
     id: 'EventActions.startImport',
-    defaultMessage: 'Start import',
-  },
-  [FINISH_IMPORT]: {
-    id: 'EventActions.finishImport',
-    defaultMessage: 'Finish import',
+    defaultMessage: 'Import',
   },
   [ASSIGN_USER]: {
     id: 'EventActions.assignUser',
```
Comment (Bam6ycha) [inline app/src/common/constants/localization/eventsLocalization.js:130]: Assign user

File: app/src/common/constants/localization/eventsLocalization.js [hunk 2]
```diff
@@ -140,14 +135,6 @@ export const actionMessages = defineMessages({
     defaultMessage: 'Unassign',
```
Comment (Bam6ycha) [inline app/src/common/constants/localization/eventsLocalization.js:135]: Unassign user

File: app/src/pages/admin/projectEventsPage/eventsEntities.jsx [hunk 3]
```diff
@@ -304,6 +320,10 @@ export class EventsEntities extends Component {
               label: intl.formatMessage(actionMessages[MATCHED_PATTERN]),
               value: MATCHED_PATTERN,
             },
+            {
+              label: intl.formatMessage(actionMessages[CREATE_PROJECT]),
```
Comment (Bam6ycha) [inline app/src/pages/admin/projectEventsPage/eventsEntities.jsx:324]: The message should be `Create project`

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/common/constants/localization/eventsLocalization.js [hunk 4]
```diff
@@ -225,8 +212,8 @@ export const actionMessages = defineMessages({
     defaultMessage: 'Delete pattern rule',
   },
   [UPDATE_AUTO_PATTERN_ANALYSIS_SETTINGS]: {
-    id: 'EventActions.updateAutoPatternAnalysis',
-    defaultMessage: 'Update pattern analysis',
+    id: 'EventActions.updateAutoPatternAnalysisSettings',
```
Comment (Bam6ycha) [inline app/src/common/constants/localization/eventsLocalization.js:215]: Should be placed after `AA changed defect type`

File: app/src/common/constants/localization/eventsLocalization.js (hunk 1 above)
Comment (BlazarQSO) [reply app/src/common/constants/localization/eventsLocalization.js:130]: Done.

File: app/src/common/constants/localization/eventsLocalization.js (hunk 2 above)
Comment (BlazarQSO) [reply app/src/common/constants/localization/eventsLocalization.js]: Done.

File: app/src/pages/admin/projectEventsPage/eventsEntities.jsx (hunk 3 above)
Comment (BlazarQSO) [reply app/src/pages/admin/projectEventsPage/eventsEntities.jsx:324]: Done.

File: app/src/common/constants/localization/eventsLocalization.js [hunk 5]
```diff
@@ -128,11 +128,11 @@ export const actionMessages = defineMessages({
   },
   [ASSIGN_USER]: {
     id: 'EventActions.assignUser',
-    defaultMessage: 'Assign',
+    defaultMessage: 'Assign user',
```
Comment (Bam6ycha) [inline app/src/common/constants/localization/eventsLocalization.js:131]: Could you please also change this message in localization files?

The same for Unassign user

Comment (BlazarQSO) [reply app/src/common/constants/localization/eventsLocalization.js:131]: Done.

**Review by Bam6ycha — APPROVED** [review]

**Pull request merged**